www/src/pages/ar/why.md

```diff
@@ -0,0 +1,52 @@
+---
+title: لماذا CT3A ؟
+description: لماذا يجب عليك اختيار Create T3 App لمشروعك المقبل
+layout: ../../layouts/docs.astro
+lang: ar
+dir: rtl
+---
+
+لقد بدأنا `create-t3-app` لان [Theo](https://twitter.com/t3dotgg) رفض أن يقوم بصنع قالب جاهز لتقنية المفضلة .  
+حبنا لـ Typescript جنبا إلى جنب مع create-next-app و [Astro's CLI](https://astro.build) هو ما الهم فريق create-t3-app للعمل على قدم وساق لنصنع أفضل نقطة بداية ممكنة لمشاريع T3.
+
+إذا كنت مهتمًا باستخدام Next.js بطريقة آمنة ، فهذا هو المكان المناسب للبدء. إذا كنت مهتمًا بأي من الخيارات التقنية المحددة التي اتخذناها ، فاقرأ :)
+
+## لماذا TypeScript ؟
+
+جافا سكريبت صعبة. لماذا نضيف المزيد من القواعد؟  
+لأننا نعتقد اعتقادًا راسخًا أن التجربة التي توفرها TypeScript ستدفعك إلى أن تكون مبرمجا أفضل، فتوفر لك الملاحظات مباشرة أثناء كتابة الكود من خلال تحديد أنواع البيانات المتوقعة(Types) ، هذا إلى جانب ميزة الإكمال التلقائي، و ينبهك في حال ارتكبت خطأَ كأن تحاول مثلا الوصول إلى خاصية غير موجودة أو تمرير قيمة من النوع الخطأ فيجبرك أن تصححها . سواء كنت مبتدئًا في تطوير الويب أو محترفًا متمرسًا ، فإن "صرامة" TypeScript ستوفر تجربة أقل إحباطًا وأكثر اتساقًا من Vanilla JS.
+
+هل لازلت غير مقتنع بـ Typesafety ؟ اذا شاهد هذا [أنت تستخدم Typescript بشكل خاطئ](https://www.youtube.com/watch?v=RmGHnYUqQ4k)
+
+## لماذا Next.js ؟
+
+نحن نحب React، بفضلها أصبح تطوير واجهات المستخدم UI سهل المنال بطريقة لم نكن نتخيلها. لقد أخذت بيد المطور وعبرت به أودية سحيقة صعبة.
+
+علي الرغم أن Next.js تقنية عنيدة قليلاَ، إلا أنها تقدم تجربة أقرب إلى الكمال في شتى النواحي routing و API حتي عرض الصور، فنحن نثق تمام الثقة أن Next.js ستقود المصورين الي الامام بقراراتها الحكيمة.
+
+## لماذا tRPC/Prisma/Tailwind وغيرها ؟
+
+بينما نؤمن بالحفاظ على بساطة الأشياء قدر الإمكان، فإن هناك قطع برمجية صغيرة تتكرر في كل تطبيق، `create-t3-app` يسمح لك بتبني هذه القطع بصورة رائعة.
+
+### لماذا tRPC ؟
+
+توفر tRPC تجربة سلسة في تطوير تطبيقات المستخدم (Client Side) كما في GraphQl، فإنها تتحايل على TypeScript لتوفر تجربة ممتازة للمطور.
```

Comment:
abuse is translated as `تتحايل`, I wonder is this fitting, because I think the intent here "using something in its full potential", as it is commonly used in video games and a like.

Do you think it's better to use a different definition?

```suggestion
توفر tRPC تجربة سلسة في تطوير تطبيقات المستخدم (Client Side) كما في GraphQl دون الحاجة لكتابة كود متداول (boilarplate)، فإنها تتحايل على TypeScript لتوفر تجربة ممتازة للمطور.